### PR TITLE
doc: Translate some part into English

### PR DESCRIPTION
**Pull request by DanSnow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,17 +47,23 @@ TODO: 詳細を書く
...
 ## CSS Only
 
+NES.css only require CSS. You can use this without JavaScript.
+
 NES.cssはCSSのみで構成されています。JavaScriptは不要です。
 
 
 ## Browser Support
 
+NES.css is compatible with the newest version of below browsers.
```
Comment (JamesIves) [inline README.md]: Proposed:

> NES.css is compatible with the newest version of the following browsers.

File: README.md [hunk 2]
```diff
@@ -44,17 +47,23 @@ TODO: 詳細を書く
 
 ## CSS Only
 
+NES.css only require CSS. You can use this without JavaScript.
```
Comment (JamesIves) [inline README.md]: Proposed:

> NES.css only requires CSS and doesn't depend on any JavaScript.

File: README.md [hunk 3]
```diff
@@ -32,9 +32,12 @@ OR
 
 TODO: npm での使用方法を書く
 
-
 ## Usage
 
+NES.css is only providing component. You can define your layout.
```
Comment (JamesIves) [inline README.md]: Proposed:

> NES.css only provides components. You're will need to define your own layout.

File: README.md [hunk 4]
```diff
@@ -32,9 +32,12 @@ OR
...
+
+Default font is `Press Start 2P`. When you using this framework with any language except English (link Japanese), please use another font as necessary.
```
Comment (JamesIves) [inline README.md]: Proposed:

> The default font is `Press Start 2P` which only supports English characters. When you're using this framework with any language except English, please use another font. 

Should `link Japanese` be an actual link?

File: README.md [hunk 5]
```diff
@@ -44,17 +47,23 @@ TODO: 詳細を書く
...
 
+NES.css is compatible with the newest version of below browsers.
+
 次のブラウザの最新バージョンをサポートしています。
 
 * Chrome
 * Firefox
 * Safari
 
+Not test
```
Comment (JamesIves) [inline README.md]: Proposed:

> Untested

File: README.md (hunk 3 above)
Comment (montezume) [reply README.md]: * You will need to define your own layout?

File: README.md (hunk 4 above)
Comment (jimmiehansson) [reply README.md]: It would be good to keep a permalink alternative available for this font

Comment (jimmiehansson) [reply README.md]: Here's the link front Dafont https://www.dafont.com/press-start-2p.font

Comment (DanSnow) [reply README.md]: That's a typo. It's "like".

**Comment by DanSnow** [comment]
@JamesIves Thank for the suggests.

I have pushed a new commit, can you please check again?

**Comment by JamesIves** [comment]
LGTM!

**Comment by Baldomo** [comment]
Hey I (mistakenly, I didn't check properly beforehand) created a whole new PR for this, although I split the README into Japanese-only and English-only

**Comment by BcRikko** [comment]
Thank you everyone 😍

**Comment by BcRikko** [comment]
@Baldomo You don't have to close #46 👍

**Pull request merged**